.travis.yml

```diff
@@ -4,8 +4,7 @@ language: ruby
 cache: bundler
 dist: trusty
 before_install:
-  - bundle -v
-  - gem install bundler -v 1.17.1
+  - gem install bundler:2.1.4
```

Comment:
is that still required for anything?

Reply:
I want to be sure the travis test is always running with the same version of bundler that the app is running